molecule/provisioner/ansible/playbooks/docker/create.yml

```
@@ -12,6 +12,10 @@
         email: "{{ item.registry.credentials.email | default(omit) }}"
         registry: "{{ item.registry.url }}"
         docker_host: "{{ item.docker_host | default(lookup('env', 'DOCKER_HOST') or 'unix://var/run/docker.sock') }}"
+        cacert_path: "{{ item.cacert_path | default((lookup('env', 'DOCKER_CERT_PATH')+'/ca.pem') if lookup('env', 'DOCKER_CERT_PATH') else omit) }}"
```

Please wrap operators with spaces:
```suggestion
        cacert_path: "{{ item.cacert_path | default((lookup('env', 'DOCKER_CERT_PATH') + '/ca.pem') if lookup('env', 'DOCKER_CERT_PATH') else omit) }}"
```

---

@webknjaz All should be done.